app/models/pub_hash.rb

```diff
@@ -148,8 +148,8 @@ def csl_doc
         cit_data_hash['event-date'] = { 'date-parts' => [[pub_hash[:conference][:year]]] } unless pub_hash[:conference][:year].blank?
         cit_data_hash['number'] = pub_hash[:conference][:number] unless pub_hash[:conference][:number].blank?
         if !pub_hash[:conference][:city].blank? || !pub_hash[:conference][:statecountry].blank?
-          cit_data_hash['event-place'] = pub_hash[:conference][:city]
-          cit_data_hash['event-place'] << ',' unless pub_hash[:conference][:city].blank? || pub_hash[:conference][:statecountry].blank?
+          cit_data_hash['event-place'] = pub_hash[:conference][:city] || ''
+          cit_data_hash['event-place'] << ',' unless cit_data_hash['event-place'].blank?
```

Comment:
This might result in an extra `,` after the city when we have a city but no state/country.